<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UPdataOrVPN</class>
 <widget class="QDialog" name="UPdataOrVPN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>更换证书和升级工具V0.0</string>
  </property>
  <widget class="QLabel" name="clientStatusLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearVpn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除VPN证书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenVpnFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开VPN证书</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>37</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="HostLineEdit">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>10</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="clientProgressBar">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>95</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVpn">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>66</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>主机IP:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开升级</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
